openedx_cas/backends.py

```diff
@@ -25,7 +79,8 @@ def auth_url(self):
 
         Redirect /CAS-provider-domain/
         """
-        raise NotImplementedError("Not implemented yet")
+        settings.CAS_SERVICE_URL = f'http://{settings.LMS_BASE}/auth/complete/{self.name}/?next=/'
```

Comment:
Don't try to override settings here. If needed, just suppose that the operator configured the setting correctly. Or define it in common.py